app/services/revocation_registry.py

```diff
@@ -425,3 +509,48 @@ async def get_credential_definition_id_from_exchange_id(
         "Successfully obtained cred definition id from the cred exchange id."
     )
     return credential_definition_id
+
+
+async def validate_rev_reg_ids(controller: AcaPyClient, request: dict):
+    """
+        Validate revocation registry ids
+
+    Args:
+        controller (AcaPyClient): aca-py client
+        request (dict): A map of revocation registry ids to a list of credential revocation ids
+
+    Raises:
+        Exception: When the revocation registry ids are invalid.
+
+    """
+    bound_logger = logger.bind(body=request)
+    bound_logger.info("Validating revocation registry ids")
+    rev_reg_id_list = list(request.keys())
+
+    if len(rev_reg_id_list) > 0:
```

Comment:
I think this if check can be dropped, because if it's an empty list the following `for` in the try just won't iterate

Or, might as well say: `if not rev_reg_id_list: return`, (i.e. if empty list just return None immediately), else: try: